src/analysis/special_functions/trigonometric.lean

```diff
@@ -1658,6 +1669,21 @@ lemma log_I : log I = π / 2 * I := by simp [log]
 
 lemma log_neg_I : log (-I) = -(π / 2) * I := by simp [log]
 
+lemma exists_pow_nat_eq (x : ℂ) {n : ℕ} (hn : 0 < n) : ∃ z, z ^ n = x :=
```

Comment:
TODO (not necessarily for this PR): generalize this to an `[is_alg_closed]` field and add an instance `[is_alg_closed complex]`.

Reply:
It would be a good idea to create an issue if you won't be working on it immediately.

Reply:
Maybe I'm misunderstanding the comment, but we did have the `complex.is_alg_closed` instance: https://github.com/leanprover-community/mathlib/blob/09afb0465df681c12c2e53d8fb88e7c6ff7e807f/src/field_theory/algebraic_closure.lean#L72